pkg/data/domain/cluster/common.go

```diff
@@ -48,7 +48,7 @@ func (s *Service) getByName(ctx context.Context, provider, name, namespace strin
 				return nil, microerror.Mask(err)
 			}
 
-		case key.ProviderOpenStack:
+		case key.ProviderOpenStack, key.ProviderCAPA, key.ProviderKVM, key.ProviderVSphere:
 			cluster, err = s.getByNameOpenStack(ctx, name, namespace)
```

Comment:
We might want to change the name of this function to something else now

Reply:
I agree. For nowt I wanted to implement a quick solution with a minimum impact. 
In general I think the process of provider detection and usage of data retrieved from Athena could use some refactoring, and I was thinking to rename the function then.
But it's also OK to do it now.

Reply:
That makes sense :) No blocker for me.